x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/search/Timestamp.java

```diff
@@ -42,14 +42,9 @@ public static Timestamp of(String milliseconds) {
         return timestamp;
     }
 
-    // time delta in nanos from other to instance
+    // time delta in nanos between this and other instance
     public long diff(Timestamp other) {
```

Comment:
This could be improved by `NANOS.between(this, other)` which behind the scenes does the math computation in this method.

Reply:
Also delta, until or between sound better than `diff`.

Reply:
Went for delta. (`until` suggests that `other` is expected to be past `this`, which is not and `between` might require a range argument)